backend/core/admin.py

```diff
@@ -25,8 +28,27 @@ class DegreeAdmin(admin.ModelAdmin):
 
 
 @admin.register(User)
-class UserAdmin(admin.ModelAdmin):
-    pass
+class UserAdmin(DjangoUserAdmin):
+    ordering = ['-date_joined']
+
+    def get_fieldsets(self, request, obj) -> List[Tuple]:
+        fieldsets = super().get_fieldsets(request, obj=obj)
+        if not obj:
+            return fieldsets
```

Comment:
This doesn't seem necessary. We might prefer it to raise an error if it happens?

Reply:
This case is actually necessary, in order to initialize the admin form used to create a new user.  
(Via the "Add User" button, or directly on http://localhost:8000/admin/core/user/add/ )